src/vowels/checkvowels.py

```diff
@@ -0,0 +1,17 @@
+def is_character_vowel(char):
+    vowels = "aeiouAEIOU"

```

Comment:
Could you please provide the documentation for it.
Since we are working on an educational project it's important to provide docstrings for each part of our code.

Reply:
"""
    Check if a given character is a vowel.

    Parameters:
    - char (str): The character to be checked.

    Returns:
    - bool: True if the character is a vowel, False otherwise.
    """

Reply:
I've added docstrings.